scripts/lib/pythonrc.py

```diff
@@ -3,6 +3,7 @@
     from django.conf import settings
     from zerver.models import *
     from zerver.lib.actions import * # type: ignore # Otherwise have duplicate imports with previous line
+    from analytics.models import *
```

Comment:
this should be conditional on `analytics` actually being an enabled app, otherwise it will crash in production :)